entrypoint.sh

```diff
@@ -60,7 +60,23 @@ export PATH=/usr/lib/go-1.10/bin:$PATH
 if [[ ${TEST_RUN} == NONE ]]; then
     echo "skipping test execution"
 else
-    go get -t -v ./...
+    CEPH_VERSION=$(dpkg -s ceph | awk '/^Version/ {print $2}')
```

Comment:
Can we pass this in from the environment or something similar? I'd like to use this same shell script on no dpkg based distros too if possible (I have private code that also runs it on fedora).

Reply:
Yes, we can! I'll update that with some changes to the Makefile and Dockerfile